examples/gno.land/p/nt/commondao/record.gno

```diff
@@ -46,20 +49,19 @@ func (r ReadonlyVotingRecord) Size() int {
 	return r.votes.Size()
 }
 
-// Choices returns the voting choices that has been voted.
-func (r ReadonlyVotingRecord) Choices() []VoteChoice {
-	choices := make([]VoteChoice, 0, r.count.Size())
-	r.count.Iterate("", "", func(k string, v any) bool {
-		choices = append(choices, VoteChoice(k))
-		return false
-	})
-	return choices
```

Comment:
Replaced in favor of an iterator which also includes the number of votes for each choice that the voting record contains